### ISSUE TITLE
QA on NE template spreadsheet

### ISSUE THREAD

**Issue by csprayregen** [pieforproviders/pieforproviders#908]
## 📋 Summary
- [x] Review template spreadsheet (@JonErvin to link)
- [x] Review documentation of spreadsheet for reference: https://docs.google.com/document/d/1IRbjS3mgbzTA8yzR38ewg10sTsfKN-HUxr9WyLpbvAw/edit
- [x] Review overview of NE rules for reference: https://docs.google.com/document/d/127Qrzf6vuA5RJ0aaAG_kfY_WBc2fNL1puwiMCnbyF38/edit
- [x] Look for any errors, bugs or other risks for broken or incorrect formulas. Discuss with @JonErvin and update as appropriate. 
- [x] Review updates with @csprayregen and @JonErvin to ensure they are implemented in each provider's spreadsheet

**Comment by csprayregen** [comment]
@joseluisrangel-dataheim tagging you here to show where you can leave feedback on the formulas

**Comment by JonErvin** [comment]
@joseluisrangel-dataheim @csprayregen Just made two updates based on the QA.  These are applied to both the [testing](https://docs.google.com/spreadsheets/d/1FKw4C6Cw0oQT6YJX8GoSZ5p4-7YaOGIxDe20tb-SaSM/edit#gid=773326511) model (the original model) and the clone for the NECC pilot.  The changes are also reflected in the [docs](https://docs.google.com/document/d/1IRbjS3mgbzTA8yzR38ewg10sTsfKN-HUxr9WyLpbvAw/edit#).

- [ ] QA Change 1: Clarified how we currently handle "aging up" mid month.  Now, the model will calculate age brackets based on the first day of the month (as you described in Slack Jose).  Therefore, a child will not "age up" until the following month.  I also added some phrasing in Table 2: Age Group in the docs to clarify that Infant is < 18 months and Toddler is >= 18 months in our model.
- [ ] QA Change 2: Added an error check in DailyCalculation tab to check if there are duplicate entries.  A duplicate entry is defined as a repeated combination of ChildID and Check-in Time.  This allows the possibility that a child may check-in and check-out multiple times in a day for some reason (not sure how common this would be).  In addition, on the dashboard I added a table to raise errors wherever we have error flags.   Right now this is for 3 things: (1) duplicate schedules, (2) duplicate attendance data, and (3) duplicate family fees.

I think covers the QA thoughts so far.  Jose and Chelsea, let me know if you're good with these changes.  And Jose, let me know if you've caught anything in the QA (thanks again!)
